src/wallet/wallet.cpp

```diff
@@ -2394,31 +2410,42 @@ bool CWallet::SetAddressBook(const CTxDestination& address, const std::string& s
 
 bool CWallet::DelAddressBook(const CTxDestination& address)
 {
-    bool is_mine;
+    isminetype is_mine;
+    const std::string& dest = EncodeDestination(address);
```

Comment:
```suggestion
    const std::string& dest{EncodeDestination(address)};
```

Reply:
Thanks but same as above, I prefer current code as it's more readable and there is no possible narrowing issue here.